src/pybind/mgr/volumes/module.py

```diff
@@ -163,6 +164,17 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'desc': "Resize a CephFS subvolume",
             'perm': 'rw'
         },
+        {
+            'cmd': 'fs nfs create',
+            'desc': "Create dummy exports",
+            'perm': 'rw'
```

Comment:
(Note: I know this is intended as an intermediate API.)

I think as an intermediate solution we want an API that sets up 1 or more exports for a number of Ganesha daemons setup outside of Ceph by the operator (i.e. vstart.sh). My understanding (cc @jtlayton) is that it should be possible to construct a Ganesha config **in RADOS** that the Ganesha daemons link to in their local `.../ganesha.conf` (similar to what you've done by linking to the `export-1`).

What that API should do is create a config for a "default" NFS gateway cluster. That would be, a nameless NFSGW cluster. The API should probably be something like:

`ceph fs nfs [cluster name] create [--fs=<fs_name>] [--recovery-pool=<pool>] [--recovery-rados-namespace=<ns>]`

By default, the command should simply use whatever file system currently exists (if and only if there is 1, otherwise raise an error that it's ambiguous). Similarly, I think the command should just create a recovery pool by default (named "ganesha-recovery"?) if the command does not specify one and one does not exist already.

When we couple this with the volumes interface, we'll use a cluster name derived from the volume/subvolume name. Same for the recovery RADOS namespace.

@jtlayton what do you think? @ajarr ?